### DISCUSSION TITLE
Welcome to HA_RFPlayerNext Discussions!

### DISCUSSION THREAD

**Discussion by crazymikefra** [crazymikefra/HA_RFPlayerNext#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Doubledom45** [comment]
Salut,
J'ai fait MAJ de ta version sur RPI et du coup Bug !
J'ai remis ancienne version, puis modifier pour moi, que j'ai mis sur mon [Github en TEST](https://github.com/Doubledom45/HA_RFPLAYER_TEST)...
@+Dom![bitmoji](https://sdk.bitmoji.com/render/panel/5b7bff30-d7cc-4141-9f91-d77fa135057f-2d67f77a-6c59-4c00-829b-9ae71d5c2ae1-v1.png?transparent=1&palette=1&width=246)
